internal/helpers/rego/config/orbs.rego

```diff
@@ -1,22 +1,31 @@
 package circleci.config
 
+import data.circleci.utils
 import future.keywords.in
 
 orbs[name] = version {
     some orb in input["orbs"]
     [name, version] := split(orb, "@")
 }
 
-ban_orbs(orb_names) = { orb_name: msg | orb_name := orb_names[_]
+default parameterized_orbs_in_input = {}
+parameterized_orbs_in_input = { orb: msg |
```

Comment:
There seems to be some indentation problems here, also I think this could be written as a pure function without the default.

It is also wild to me to see that you wrote it as a rule but are still able to invoke it as a function, but that's rego for you! very flexible